qlib/kernel/SignalDef.rs

```diff
@@ -89,6 +87,45 @@ impl PtRegs {
         self.rsp = ctx.rsp;
         self.ss = ctx.ss as u64;
     }
+
+    pub fn get_stack_pointer(&self) -> u64 {
+        return self.rsp;
+    }
+
+    pub fn set_stack_pointer(&mut self, sp: u64) {
+        self.rsp = sp;
+    }
+}
+
+#[cfg(target_arch = "aarch64")]
+#[repr(C)]
+#[derive(Debug, Copy, Clone, Default)]
```

Comment:
Could you add a custom Debug trait for aarch64 SigContext?

The __reserved field doesn't not need to be printed, and it is too large, making the log very noisy. (4k lines printed per run).  The following snippet would work:

```rust
#[cfg(target_arch = "aarch64")]
impl fmt::Debug for SigContext {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        f.debug_struct("SigContext")
            .field("fault_address", &self.fault_address)
            .field("Regs", &self.regs)
            .field("sp", &self.sp)
            .field("pc", &self.pc)
            .field("pstate", &self.pstate)
            .finish()
    }
}

```

Reply:
@shrik3 I can create a patch for this.

Reply:
done